exchanges/btse/btse_wrapper.go

```diff
@@ -397,7 +415,32 @@ func (b *BTSE) GetFundingHistory() ([]exchange.FundHistory, error) {
 
 // GetExchangeHistory returns historic trade data within the timeframe provided.
 func (b *BTSE) GetExchangeHistory(p currency.Pair, assetType asset.Item, timestampStart, timestampEnd time.Time) ([]exchange.TradeHistory, error) {
-	return nil, common.ErrNotYetImplemented
+	if assetType != asset.Spot {
+		return nil, common.ErrNotYetImplemented
```

Comment:
Since futures was already half implemented and the only difference is a /spot to /futures i have gone ahead and implemented ticker/orderbook for futures